Droidcon-Boston/app/src/main/res/layout/agenda_detail_fragment.xml

```diff
@@ -17,6 +17,16 @@
             android:layout_marginBottom="16dp"
             android:orientation="horizontal">
 
+            <ImageButton
+                    android:id="@+id/image_back"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:padding="8dp"
+                    android:layout_gravity="center_vertical"
+                    android:contentDescription="@string/hint_bookmark"
```

Comment:
probably needs new hint text instead of telling users this is a "bookmark" :-)

Reply:
updated